src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -403,7 +483,7 @@ private SafeTrainingAndModelBuffer PrepareBuffer()
         /// <param name="validationData">The validation data set.</param>
         public MatrixFactorizationPredictionTransformer Train(IDataView trainData, IDataView validationData = null)
         {
```

Comment:
Can we make this private? Or internal?

Reply:
No, this one needed for the case when you train with validation data.

---
In reply to: [255196397](https://github.com/dotnet/machinelearning/pull/2451#discussion_r255196397) [](ancestors = 255196397)

Reply:
You are right, but this seems so strange! Don't you think?

---
In reply to: [255197330](https://github.com/dotnet/machinelearning/pull/2451#discussion_r255197330) [](ancestors = 255197330,255196397)

Reply:
My bad thank you!

---
In reply to: [255197330](https://github.com/dotnet/machinelearning/pull/2451#discussion_r255197330) [](ancestors = 255197330,255196397)